static/app/views/replays/replayTable/index.tsx

```diff
@@ -53,10 +54,12 @@ function ReplayTable({
   emptyMessage,
   saveLocation,
   gridRows,
+  footer,
 }: Props) {
   const routes = useRoutes();
   const newLocation = useLocation();
   const organization = useOrganization();
+  const showBottomBorder = visibleColumns.includes(ReplayColumn.MOST_RAGE_CLICKS);
```

Comment:
should this also check to see if `MOST_ERRONEOUS_REPLAYS` is included?

Reply:
this was an accidental change -- it's relating to the alerts styling, but it's showing green here because i moved its position in the file🫥